### DISCUSSION TITLE
Domain amazon installation doesn't work with Mozilla hubs (ROLLBACK_COMPLETE)

### DISCUSSION THREAD

**Discussion by Dayk0** [mozilla/hubs#3168] [Hubs Cloud Questions] [answered]
Hello,

I have an installation problem to install mozilla on my domain name. I bought a domain name on amazon route 53, followed all the indications in the youtube tutorials but nothing to do, I tried 3 times to reinstall by deleting my stack but still the same. I really do not know what to do.
Should I create other domain names? I have tried to follow some bug reports indicated on the site but it does not work (https://hubs.mozilla.com/docs/hubs-cloud-aws-troubleshooting.html)
Here is a screenshot of the different errors:

http://snap.ashampoo.com/uploads/2020-10-14/F87q7S0aOIt2YLtuWA8y9du3RoFXkAfq0Db7wA7gJRBd7UxvVVOUYDQLsWe6XJNX.png

http://snap.ashampoo.com/uploads/2020-10-14/JfRe0uEqmmtVrfMSH9T9MwT62STYtQm7w8cwthhat2wFAeyIdniPe4NaPCYDtFBK.png

I greatly thank whoever manages to help me install this server!

See you soon. :)

**Comment by robinkwilson** [comment, marked as answer]
Hello let's get you unstuck! 

Which domain recipe did you follow? [Hubs Docs Domain Recipes](https://hubs.mozilla.com/docs/hubs-cloud-aws-domain-recipes.html) You'll need at least 2 domains to get your stack installed. 

I need the _first_ error message on your rollback. Scroll ALL THE WAY DOWN to find the first red error with a name to the error. After that, try searching for the error name in the guide: [Hubs Docs AWS Troubleshooting](https://hubs.mozilla.com/docs/hubs-cloud-aws-troubleshooting.html)

You may have to retry the stack deploy with the same params 2x, since it's a complicated process sometimes something goes wrong the first time. 

Thanks! Let me know how it goes.

**Reply by robinkwilson** [reply]
**[Temporarily marked as answered]** Unmark this as "answered", when you add more information.

**Reply by Dayk0** [reply]
Ah ok, I didn't buy two domains but only one. Maybe that's why it doesn't work. I will try again by purchasing another domain name. Here are all the errors: http://snap.ashampoo.com/RXzGfZULUqXFaRgVeS3ey2Yn4xgNSWbFP2mUVcblsYrsLxIqlfueAoR5gXlzPM0a
